components/brave_rewards/resources/page/components/tips_panel.tsx

```diff
@@ -121,19 +130,30 @@ export function TipsPanel () {
     return (
       <>
         <ConfigHeader />
-        <style.description>
-          {getString('donationAbility')}
-        </style.description>
-        {
-          Object.keys(data.inlineTip).map((site) => (
-            <PanelItem key={site} label={lookupPublisherPlatformName(site)}>
-              <ToggleButton
-                checked={data.inlineTip[site]}
-                onChange={toggleHandler(site)}
-              />
-            </PanelItem>
-          ))
-        }
+        <PanelItem
+          label={getString('donationAbility')}
+          separateChildren={true}>
```

Comment:
Is a separate prop necessary if we only ever pass `true`?

Reply:
This prop is an optional (https://github.com/brave/brave-core/pull/17339/files?show-viewed-files=true&file-filters%5B%5D=#diff-2dd3b1227860d25af473b0305ddb96607bf6f7802c57b1439b7b890406e817e3R90) so in all other uses of `PanelItem` it should retain the current behavior of the control. Or is there a better way to do this?

Reply:
Gotcha, I didn't catch that it was optional.